src/components/menu-bar/menu-bar-menu.css

```diff
@@ -0,0 +1,7 @@
+@import "../../css/units.css";
+@import "../../css/z-index.css";
+
+.menu-bar-menu {
+    z-index: $z-index-menu-bar;
+    top: $menu-bar-height;
```

Comment:
Positioning styles for children should be done by the parent component.

Reply:
I see the logic of that. I created a container div that applies the position via margin-top instead.